docs/userguide/packaging/index.md

```diff
@@ -76,12 +78,14 @@ next version of ICU is installed.
 Modify 'uconfig.h' to add the following lines before the first #ifndef
 UCONFIG_... section
 
+```c
```

Comment:
With the explicit code block, consider un-indenting the code.

Reply:
done